contributing_to_docs/doc_guidelines.adoc

```diff
@@ -911,22 +911,134 @@ kind: Pod
 metadata:
   labels:
     test: liveness
-...
+# ...
```

Comment:
Is this something new that you're proposing to try to get rid of that highlighting issue YAML sometimes has with these? I don't think I've seen this done anywhere in the repo yet.

And if we are going to do that, I'd update the intro sentence for these examples to specifically say to use # ahead of the ellipses. Or if it's only for YAML that we'd do that, specify that.

Reply:
@bergerhoffer, in YAML, `...` is invalid. By prefacing it with a comment `#` which is valid, the YAML becomes valid. Someone could then paste it into an editor or where ever and not need to manually delete instances of `...` to have valid YAML.